motoman_gp20hl_support/urdf/gp20hl_macro.xacro

```diff
@@ -146,7 +146,7 @@
       <origin xyz="${0 + kinematic_offsets['6_t']['x']} ${-0.1 + kinematic_offsets['6_t']['y']} ${0 + kinematic_offsets['6_t']['z']}"
               rpy="${radians(-90) + kinematic_offsets['6_t']['roll']} ${0 + kinematic_offsets['6_t']['pitch']} ${0 + kinematic_offsets['6_t']['yaw']}"/>
       <axis xyz="0 0 1"/>
-      <limit lower="${radians(-455)}" upper="${radians(455)}" effort="19.6" velocity="${radians(700)}"/>
+      <limit lower="${radians(-210)}" upper="${radians(210)}" effort="19.6" velocity="${radians(700)}"/>
```

Comment:
```suggestion
      <limit lower="${radians(-455)}" upper="${radians(455)}" effort="19.6" velocity="${radians(630)}"/>
```
to match the datasheet -- can we do a quick verification on the other joints as well? https://www.motoman.com/en-us/products/robots/industrial/assembly-handling/gp-series/gp20hl

Reply:
there are apparently two different models of the GP20HL with varying joint limits, this is correct for what we have